pkg/cmd/cli/nodeagent/server.go

```diff
@@ -93,6 +97,7 @@ func NewServerCommand(f client.Factory) *cobra.Command {
 		metricsAddress:          defaultMetricsAddress,
 		resourceTimeout:         defaultResourceTimeout,
 		dataMoverPrepareTimeout: defaultDataMoverPrepareTimeout,
+		profilerAddress:         defaultProfilerAddress,
```

Comment:
Since this is going to a patch of v1.14.x can we not enable the profiler by default?  
We may ask the user to patch the spec of node-agent to turn on the profiler when needed.

Reply:
Done, removed the default value